ui/src/views/Settings.vue

```diff
@@ -3,45 +3,110 @@
   SPDX-License-Identifier: GPL-3.0-or-later
 -->
 <template>
-  <cv-grid fullWidth>
-    <cv-row>
-      <cv-column class="page-title">
+  <div class="bx--grid bx--grid--full-width">
```

Comment:
Why were `<cv-grid>`, `<cv-row>` and `<cv-column>` removed and replaced with `<div>`? :cry:

Reply:
oups....my bad sorry